src/tests/multihost/basic/conftest.py

```diff
@@ -42,7 +42,7 @@ def package_install(session_multihost):
     if 'Fedora' in distro:
         cmd = 'dnf install -y %s' % (pkg_list)
     elif '8.' in distro.split()[5]:
-        cmd = 'dnf module -y install idm:4 389-ds:1.4'
+        cmd = 'dnf module -y install 389-ds:1.4'
```

Comment:
This seems like unrelated to the rest of the commit, should this change be split into a different commit?